content/pages/index.md

```diff
@@ -15,6 +15,12 @@ The recommended package for reading and writing Excel 2010 files (ie: .xlsx)
 
 [Download](http://pypi.python.org/pypi/openpyxl) | [Documentation](https://openpyxl.readthedocs.org/) | [Bitbucket](https://bitbucket.org/openpyxl/openpyxl)
 
+### pyexcel
+
+Library with a unified api for reading and writing files with older Excel files (xls),  Excel 2010 files (xlsx), Open Spreadsheet files (ods), and many other data formats. 
```

Comment:
api -> API?

Not 100% it belongs on this site, though, since it appears to just be a bunch of wrappers around other libraries, such as `xlrd`, `xlwt`, etc...

Reply:
> Not 100% it belongs on this site, though, since it appears to just be a bunch of wrappers around other libraries

I don't quite see why this is a problem, it's a library for interfacing with excel files, regardless of how it achieves that. Since python-excel.org is the first result when searching "python excel libraries" it would be nice to let people know that there are options for interfacing with both formats without writing your open wrapper or duplicating logic.

Maybe it can go in a different spot on the page and we adjust the language to clarify that it is a wrapper around openpyxl, xlrd, etc that uses a unified ~~api~~API. Could also include pandas, but I don't believe pandas can write xls files.

Reply:
Sure, probably a newer section, "Higher level libraries and frameworks" or some such. Anyway, it's moot until the publishing process is fixed.

Reply:
@cjw296 I'll take a look at what's gone wrong with the GH action